rotkehlchen/utils/misc.py

```diff
@@ -315,3 +326,9 @@ def rgetattr(obj: Any, attr: str, *args: Any) -> Any:
     def _getattr(obj: Any, attr: str) -> Any:
         return getattr(obj, attr, *args)
     return functools.reduce(_getattr, [obj] + attr.split('.'))
+
+
+def pairwise(iterable: Iterable[Any]) -> Iterator:
+    "s -> (s0, s1), (s2, s3), (s4, s5), ..."
+    a = iter(iterable)
+    return zip(a, a)
```

Comment:
good one :)

Reply:
was already in the app. Just ... only in the cryptocompare module :sweat_smile: